pkg/kubelet/lifecycle/predicate.go

```diff
@@ -128,7 +128,7 @@ func (w *predicateAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 		switch re := r.(type) {
 		case *predicates.PredicateFailureError:
 			reason = re.PredicateName
-			message = re.Error()
+			message = fmt.Sprintf("predict %s failed, because %s", reason, re.GetReason())
```

Comment:
This change does not make sense to me.

There's no need to rename `Error()` interface.

And, `reason` has already been extracted in every case and returned to podAdmitHandler of kubelet, and eventually it will reject pod with both `reason` & `message`:

```
kl.statusManager.SetPodStatus(pod, v1.PodStatus{
		Phase:   v1.PodFailed,
		Reason:  reason,
		Message: "Pod " + message})
```

Reply:
revert this snippet, please take another look